Automattic/wp-calypso
# Activity log: Add empty period message

Adds an empty notice when there are no logs to display.

~Note that the empty component is essentially a stub right now, I'm hoping to get some design and or copy feedback. That could happen in this PR or after.~

Apologies, this PR _should_ be small, but I accidentally committed a prettier format (#17039) 🙁 

## Testing
1. Visit https://calypso.live/stats/activity?branch=add/activitylog-placeholder&startDate=2017-07-01
1. Ensure you're at an empty view (navigating a month back will work ATM).
1. You should see the notice.
1. Ensure that loading state does not show empty notice.
1. Ensure that views _with_ logs are unaffected.

## Screens

[updated capture](https://github.com/Automattic/wp-calypso/pull/17032#issuecomment-321246943)

![no-activity](https://user-images.githubusercontent.com/841763/29118151-1b4dcb00-7d01-11e7-95b8-0495fd885b33.png)

Fixes #14662
